### ISSUE TITLE
LMB in ui_accept no longer working with OptionButton

### ISSUE THREAD

**Issue by esap120** [godotengine/godot#64578]
### Godot version

3.5 stable

### System information

Windows 10, GLES3

### Issue description

Possible regression where having left mouse button under the "ui_accept" input action breaks the OptionButton since the popup menu no longer displays. It works fine in 3.4.5 but in 3.5 it now seems to swallow the mouse input. If I remove LMB from ui_accept it works again.

This could be happening with other controls, but this is the one I have noticed. Clicking still focuses the button but the popup does not appear.

Setting `Input.set_use_accumulated_input(false)` does not change the result, LMB will still not work

### Steps to reproduce

Create a scene with an OptionButton and add Left Mouse Button to "ui_accept" under InputMap in project settings. Try to mouse click on OptionButton to get an popup menu to appear

### Minimal reproduction project

[test_project.zip](https://github.com/godotengine/godot/files/9375111/test_project.zip)

**Comment by Calinou** [comment]
@esap120 Can you reproduce this in any of the [3.5 betas and RCs](https://downloads.tuxfamily.org/godotengine/3.5/) to determine when the regression started?

**Comment by esap120** [comment]
@Calinou looks like since RC 1 after trying them out. Could be related to the PRs that made improvements to the OptionButton:

https://github.com/godotengine/godot/pull/61102
https://github.com/godotengine/godot/pull/61185

I'll see if I can build from source to confirm though

**Comment by esap120** [comment]
I was able to build from source and after trying out a bunch of builds it looks like this is the PR that caused the regression: https://github.com/godotengine/godot/pull/57643 specifically this section of code: https://github.com/godotengine/godot/pull/57643/files#diff-f53516902cc0ae0bcdff8847d815c5644836b1fc138517b47b6cb5f0b807ae38R114-R119

**Comment by Calinou** [comment]
cc @YeldhamDev

**Comment by esap120** [comment]
@YeldhamDev I don't write much c++ or do any godot engine development but would something like this make sense?: https://github.com/godotengine/godot/compare/3.x...esap120:godot:mouse-button-fix

Basically just inspecting if the ui_accept action is an InputEventMouseButton in _gui_input.

Also I might need to update my sample project, I noticed when testing with the visual studio debugger and running the test scene directly it would use the default InputMap actions and not the ones defined in the project. I add a script to the scene that manually adds the lmb press and that made it easier to test:

```
var left_click: InputEventMouseButton = InputEventMouseButton.new()
left_click.factor = 1
left_click.button_index = 1
left_click.button_mask = 0
InputMap.action_add_event("ui_accept", left_click)
```

**Comment by YeldhamDev** [comment]
@esap120 I'm already planning a fix that should hopefully be less hacky than that. I just need a little more time.

**Comment by akien-mga** [comment]
Fixed by #64636.
